crates/astria-sequencer/src/proposal/commitment.rs

```diff
@@ -3,6 +3,24 @@ use std::collections::BTreeMap;
 use bytes::Bytes;
 use proto::native::sequencer::v1alpha1::SignedTransaction;
 
+/// Wrapper for values returned by [`generate_sequence_actions_commitment`].
+pub(crate) struct GenerateCommitmentsResult {
```

Comment:
Maybe leave out the `Result` suffix to not involve memories of the Result type. Just `GeneratedCommitments` is fine

Also, do we need to make this a public interface at some point given that it's returned in the ABCI interface below?

Reply:
don't need to make it public, it's not returned publicly, this struct is just used to populate the txs returned